File: netease-nimsdk/src/main/java/com/zxn/netease/nimsdk/business/session/viewholder/media/DateViewHolder.java
package com.zxn.netease.nimsdk.business.session.viewholder.media;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.zxn.netease.nimsdk.R;

/**
 * Created by winnie on 2017/9/18.
 */

public class DateViewHolder extends RecyclerView.ViewHolder {

    public TextView dateText;

    public DateViewHolder(View itemView) {
        super(itemView);
        dateText = itemView.findViewById(R.id.date_tip);
    }
}

File: netease-nimsdk/src/main/java/com/zxn/netease/nimsdk/business/session/viewholder/media/DateViewHolder.kt
package com.zxn.netease.nimsdk.business.session.viewholder.media

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.zxn.netease.nimsdk.R

/**
 * 日期.
 */
class DateViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @JvmField
    var dateText: TextView

    init {
        dateText = itemView.findViewById(R.id.date_tip)
    }
}